coralproject/talk
# update bad username copy

## What does this PR do?

updates the error when a user tries to make a username with profanity in it.

## How do I test this PR?

- check your list of naughty words
- try to make a username that violates the policy
- you'll see the new error
